anvill/python/anvill/binja/bnprogram.py

```diff
@@ -34,7 +34,11 @@ def __init__(self, bv: bn.BinaryView, path: str):
         self._path: Final[str] = path
         self._bv: Final[bn.BinaryView] = bv
         self._type_cache: Final[TypeCache] = TypeCache(self._bv)
-        self._init_ctrl_flow_redirections()
+
+        try:
+            self._init_ctrl_flow_redirections()
+        except:
+            print("Failed to initialize control flow redirections")
```

Comment:
File an issue that we eventually need a better logging approach.

Reply:
Opened it here: https://github.com/lifting-bits/anvill/issues/166